src/WebJobs.Script.WebHost/Extensions/TaskExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Extensions
+{
+    public static class TaskExtensions
```

Comment:
Class name confused me at first. This is really an IEnumerable of Task extensions class

Reply:
done